net/instaweb/rewriter/server_context.cc

```diff
@@ -1357,7 +1357,7 @@ GoogleString ServerContext::ShowCacheForm(StringPiece user_agent) {
       "  URL: <input id=metadata_text type=text name=url size=110 /><br>\n"
       "  User-Agent: <input id=user_agent type=text size=103 name=user_agent ",
       ua_default,
-      "/></br> \n",
+      "/><br> \n",
```

Comment:
I think the right thing to do here would be to move the slash so this ends up like `<br/>`?

Reply:
The `/` character is optional for <br> (<br/>). But if you like `<br/>` more than `<br>` then we should do that! :)

Reply:
:smiley: